app/classes/document_query.rb

```diff
@@ -25,34 +33,48 @@ def initialize(options)
     @ignore_tags = options[:ignore_tags]
     @date_range = { gte: @options[:min_timestamp], lt: @options[:max_timestamp] }
     @search = Search.new
-    @included_sites, @excluded_sites = [], []
+    @included_sites = []
+    @excluded_sites = []
     parse_query(options[:query]) if options[:query]
   end
 
   def body
-    search.source source_fields
+    search.source(source_fields)
```

Comment:
The method calls in this file that use the Elasticsearch DSL should actually not include parentheses around the args. They fall under the rule "Always omit parentheses for methods that are part of an internal DSL" (https://rubystyle.guide/#methods-that-are-part-of-an-internal-dsl). Please:
- back out the new parentheses
- update .rubocop.yml to exclude this file from the `Style/MethodCallWithArgsParentheses` cop, with a comment explaining why

Reply:
Parens returned and `.rubocop.yml` updated.